ui/src/submissions/seminars/components/SeminarForm.jsx

```diff
@@ -72,7 +69,9 @@ function SeminarForm({ values, isSubmitting, isValid, isValidating }) {
                 placeholder="Family name, first name"
                 pidType="authors"
                 suggesterName="author"
-                extractItemCompletionValue={getSuggestionSourceNameValue}
+                extractItemCompletionValue={
```

Comment:
you should leave `extractItemCompletionValue` as it is. Because it's what is filled in the form and sent as author name to the backend. We don't wanna send `Urhan, Harun (CERN)` as the author name. Institution should be only for display.

That's why you should this function pass `renderResultItem` prop instead.